WISE-Community/WISE
# Added support for i18n in code

Added support for i18n in code and updated the git commit hooks to extract i18n messages not in templates and merge them into the messages.xlf file.

Added messages for text defined in component .ts files as well as translations for html attributes. 

To test:
- Load home, about, features, contact pages and make sure all text appears correctly.
- Log in as student and teacher and make sure all text appears correctly. 
- Go through the forgot password and forgot username wizards for both students and teachers and make sure they work.
- Make sure messages in code show up in the messages.xlf file.

Closes #1683.
